### DISCUSSION TITLE
Not working .setTexture on materials channel if model viewer is not on screen.

### DISCUSSION THREAD

**Discussion by Redsky48** [google/model-viewer#4019] [Q&A]
### Description

If i try to set texture on materials channel while model viewer is in view on my screen, than it is working. But if i minimize screen or scroll down so it is not visible or make it display none, it does not work anymore.

And also could it be possible to add  code bellow to ControlsModelViewerElement? So it would be easier to set field of view? Thank you.

```
setFieldOfView(fov) {
	return this[$controls].setFieldOfView(fov);
}
```

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: v2.1.1

### Browser Affected

- [x] Chrome
- [x] Edge
- [x] Firefox
- [x] IE
- [x] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

### AR

- [ ] WebXR
- [x] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Can you post a repro link (the one you left in the template is unedited)? And perhaps a screen recording of what "doesn't work" looks like? Any errors in the console log?

As with all HTML elements, the attributes all have JS setters, so `field-of-view="25deg"` becomes `mv.fieldOfView="25deg";`. Will that work?
